cargo-frc/src/config.rs

```diff
@@ -91,11 +93,21 @@ pub fn get_config() -> Result<FrcConfig, String> {
         error!("Neither a team number or rio address was specified.");
     };
 
+    debug!("year val: {:?}", frc.get("year"));
+    // debug!("team-number val: {:?}", frc.get("team-number"));
+
+    let toolchain_year = frc
+        .get("year")
+        .and_then(|v| v.as_u64())
+        .map(|i| i.to_string())
+        .and_then(|s| Toolchain::from_year(s.as_str()));
```

Comment:
Instead of converting an integer to a string then effectively back, maybe `from_year` should take an integer instead.